pkg/minishift/registration/utils.go

```diff
@@ -27,9 +27,6 @@ import (
 type RegistrationParameters struct {
 	Username      string
 	Password      string
-	Proxy         string
-	ProxyUser     string
-	ProxyPassword string
```

Comment:
Since for subscription we are not using individual flag for proxy so removing those variable from struct.